src/lib.rs

```diff
@@ -59,19 +67,20 @@ impl Builder {
     ///
     /// The max level is the [max_level](Self::max_level) field.
     pub fn init(self) -> Result<(), SetLoggerError> {
-        log::set_logger(&EguiLogger).map(|()| log::set_max_level(self.max_level))
+        log::set_max_level(self.max_level);
+        log::set_boxed_logger(Box::new(self.build()))
```

Comment:
This _should_ fix CI, yet the Issue that this PR fixes will still apply to wasm (or other no_std environments) 
```suggestion
        #[cfg(all(feature = "std", target_has_atomic = "ptr"))]
        log::set_boxed_logger(Box::new(self.build()))?;
        Ok(())
```
It might be smart to add a `std` feature to this crate for future cases.
I think someone _fixed_ wasm before. 
I might take a look at better wasm support in the future though